sn_client/src/api.rs

```diff
@@ -849,17 +896,25 @@ impl Client {
 
         // When there is retry on Put side, no need to have a retry on Get
         let verification_cfg = GetRecordCfg {
-            get_quorum: Quorum::Majority,
+            get_quorum: Quorum::All,
```

Comment:
> as there will always be one node failes the get verification

Why? 

Right now with `GetRange` all == `CloseGroupSize` as opposed to all returners... so it allows for stricter early exit.

Really though it's not so much about `All` so much as there being a difference in the checking between `verify` or not... So it could be `majority` in both cases and we just remove the `verify` flag on the fn perhaps? (Which I do note is as main... but it seemed handy to have this distinction to me...)